### PR TITLE
Switch up chapter 1: make it more engaging and fun.

### PR DESCRIPTION
**Pull request by mosoriob**
I switched up Chapter 1. It's too wordy and formal. Let's make it more engaging and fun.

### CODE REVIEW

**Comment by mosoriobnew** [comment]
I love it! ❤️

**Comment by yolandagil** [comment]
I also like this change!

**Comment by likhitjha** [comment]
**Ohh nice**

**Comment by dborrell123** [comment]
Looks great!

**Comment by 18dhruvm** [comment]
Looks fab!

**Review by mosoriobnew — APPROVED** [review]
